src/Playwright/Core/CDPSession.cs

```diff
@@ -45,7 +45,8 @@ internal override void OnMessage(string method, JsonElement? serverParams)
         switch (method)
         {
             case "event":
-                OnCDPEvent(serverParams!.Value.GetProperty("method").ToString(), serverParams.Value.GetProperty("params"));
+                serverParams!.Value.TryGetProperty("params", out var cdpParams);
+                OnCDPEvent(serverParams!.Value.GetProperty("method").ToString(), cdpParams);
```

Comment:
Since `JsonElement` is a struct it might be nicer to pass `null` instead of a `default(JsonElement)`? Looks like `OnCDPEvent` accepts `null`.

```suggestion
                OnCDPEvent(serverParams!.Value.GetProperty("method").ToString(), serverParams!.Value.TryGetProperty("params", out var cdpParams) ? cdpParams : null);
```
(not sure if that compiles)